packages/optimistic-oracle/test/proposer.js

```diff
@@ -441,6 +458,33 @@ contract("OptimisticOracle: proposer.js", function(accounts) {
     assert.isTrue(spyLogIncludes(spy, -1, "sendDisputes"));
   });
 
+  it("Cannot set disputePriceErrorPercent < 0 or >= 1", async function() {
+    try {
+      new OptimisticOracleProposer({
+        logger: spyLogger,
+        optimisticOracleClient: client,
+        gasEstimator,
+        account: botRunner,
+        ooProposerConfig: { disputePriceErrorPercent: -0.1 }
+      });
+      assert(false);
+    } catch (err) {
+      assert(true);
+    }
+    try {
+      new OptimisticOracleProposer({
+        logger: spyLogger,
+        optimisticOracleClient: client,
+        gasEstimator,
+        account: botRunner,
+        ooProposerConfig: { disputePriceErrorPercent: 1 }
```

Comment:
I can see why, practically speaking, you would never want a > 100% margin of error, but is there a technical reason for this as well?

Reply:
I'm gonna remove the base and quote nomenclature but I'll make it more clear that for this specific test it's just arbitrary and I just need to initiate these values

Reply:
I suppose there is technically no limit....do you think we should allow it? Curious what @mrice32 @daywiss think too